public/doc/openapi-3-v1.2.json

```diff
@@ -4295,9 +4295,6 @@
       },
       "OrderProcess": {
         "type": "object",
-        "required": [
```

Comment:
Name may be required for creation. How do we distinguish constraints for creation and update in openapi model?

Reply:
We don't. We don't do that for portfolios or portfolio items either, we simply let the API return an error if they do not supply a name. I figured since that's how we handle it for portfolios/portfolio items, that it's fine to handle it this way for order processes as well.